integration/rocket-chat/main.js

```diff
@@ -517,6 +517,9 @@ function SmartiWidget(element, _options) {
      * @constructor
      */
     function IrLatchWidget(params,wgt_conf) {
```

Comment:
This implementation encapsulates search engine specific implementations details. It would be more elegant to introduce a full-text search widget that abstracts those particularities. Somewhere in the future this "jquery"-style `main.js` has to refactored in general. So I accept the ugliness for now.